src/app/src/mainwindow.cpp

```diff
@@ -230,19 +233,54 @@ void MainWindow::on_ledIntensityEdit_valueChanged(double value) {
 }
 
 
+/*
+ * Check if drive on windows has sufficient space for acquisition.
+ *
+ * @param drive letter selected in config file
+ * @param fps setting of acquisition
+ * @param duration of acquisition
+ * @param TODO
+ *
+ * @returns boolean true if space is available
+*/
+bool available_space_in_drive(std::filesystem::path driver_name, double fps,double duration,double frameBytes){
```

Comment:
you probably want to put this entire function in a
```
#ifdef _WIN32
...
#endif
```
since it can only compile on windows